archunit/src/main/java/com/tngtech/archunit/core/domain/DomainObjectCreationContext.java

```diff
@@ -143,6 +143,14 @@ public static Source createSource(URI uri, Optional<String> sourceFileName) {
         return ThrowsClause.from(codeUnit, types);
     }
 
+    public static Set<InstanceofCheck> createInstanceofCheck(JavaCodeUnit codeUnit, List<JavaClass> types) {
```

Comment:
I think this method should use the plural, i.e. be called `createInstanceofChecks(..)`, since it creates a set of `InstanceofCheck`